driver-sync/src/test/functional/com/mongodb/client/CrudProseTest.java

```diff
@@ -312,15 +313,6 @@ private void assertBulkWriteHandlesCursorRequiringGetMore(final boolean transact
         }
     }
 
-    private static Stream<Arguments> testBulkWriteErrorsForUnacknowledgedTooLargeInsertArgs() {
-        return Stream.of(
-                arguments("insert", false),
-                arguments("insert", true),
-                arguments("replace", false),
-                arguments("replace", true)
-        );
-    }
-
```

Comment:
This was a leftover from https://github.com/mongodb/mongo-java-driver/pull/1495/commits/ee3073ec292f49dcaf202c3aca29bc29668fa064.